source/includes/_upload.md

```diff
@@ -1,68 +1,88 @@
+curl -X POST "https://siasky.net/skynet/skyfile" -F file=@image.jpg
+```
+
+```javascript
+const skynet = require('@nebulous/skynet');
+
+(async () => {
+	const skylink = await skynet.UploadFile(
+		"./image.jpg",
+		skynet.DefaultUploadOptions
+	);
+	console.log(`Upload successful, skylink: ${skylink}`);
+})();
 ```
 
 ```python
 from siaskynet import Skynet
-response = Skynet.upload_file("image.jpg")
-print(responsek)
-```
 
-```javascript
-import { upload } from "skynet-js";
-const response = await upload(portalUrl, file);
-console.log(response)
+skylink = Skynet.upload_file("image.jpg")
+print("Upload successful, skylink: " + skylink)
 ```
 
 ```go
 package main
 
 import (
-	skynet "github.com/NebulousLabs/go-skynet"
 	"fmt"
+	skynet "github.com/NebulousLabs/go-skynet"
 )
-                        
+
 func main() {
-	response, err := skynet.UploadFile("image.jpg", skynet.DefaultUploadOptions)
+	skylink, err := skynet.UploadFile("./image.jpg", skynet.DefaultUploadOptions)
 	if err != nil {
-		panic(err)
+		fmt.Println("Unable to upload:", err.Error())
+		return
 	}
-	fmt.Printf("%+v", response)
+	fmt.Printf("Upload successful, skylink: %v\n", skylink)
 }
 ```
 
-> The above command returns JSON structured like this
+Uploading a file to Skynet can be done through a Skynet portal or your
+local siad instance. All SDKs follow the pattern `skynet.Upload(path,
+Settings)` as much as possible.
 
-```json
+### Upload Settings
+
+Field      | Description
+---------- | -----------
+`portal`   | The URL of the portal
+`filename` | Custom filename. This is the filename that will be returned when downloading the file in a browser.
+
+### Upload Response
+
+```shell
 {
-"skylink": "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg",
-"merkleroot": "QAf9Q7dBSbMarLvyeE6HTQmwhr7RX9VMrP9xIMzpU3I",
-"bitfield": 2048
+  "skylink": "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg",
+  "merkleroot": "QAf9Q7dBSbMarLvyeE6HTQmwhr7RX9VMrP9xIMzpU3I",
+  "bitfield": 2048
 }
 ```
 
-Uploading a file to Skynet can be done through a Skynet portal or your
-local siad instance. All SDKs follow the pattern `skynet.Upload(path,
-Settings)` as much as possible. The `Settings` 
+```javascript
+// skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg"
+```
 
-### Upload Settings
+```python
+# skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg"
+```
 
-Field     | Description
---------- | -----------
-portal    | the URL of the portal
-filename  | custom filename, this is the filename that will be returned when downloading the file in a browser
+```go
+// skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg"
+```
 
-### Upload Response
+The response will contain some or all of these fields:
 
-Field     | Description
---------- | -----------
-skylink   | `TODO`
-merkleroot | `TODO`
-bitfield | `TODO`
+Field        | Description
+------------ | -----------
+`skylink`    | This is the skylink that can be used when downloading to retrieve the file that has been uploaded.
```

Comment:
Could use a more technical explanation, see`modules/skylink.go` to read up on it. For sure it should mention it's a 46 character base64 encoded string that represents uploaded content and can be used to access that contnet.

Reply:
Added some more detail